CHANGES.md

```diff
@@ -24,6 +24,10 @@ OpenSSL 3.1
 
 ### Changes between 3.0 and 3.1 [xx XXX xxxx]
 
+ * Added the ability for BIO_s_mem() to read/write datagrams
```

Comment:
The changes entry should be slightly adjusted to reflect it is now BIO_s_dgram_mem().

Reply:
Fixed